app/classifier/mini-course.cjsx

```diff
@@ -2,7 +2,7 @@ React = require 'react'
 createReactClass = require 'create-react-class'
 Dialog = require 'modal-form/dialog'
 Translate = require 'react-translate-component'
-MediaCard = require '../components/media-card'
+`import MediaCard from '../components/media-card';`
```

Comment:
Not a blocking comment, but backticked imports break tests, so I prefer `require().default` to import ES6 modules in coffeescript.

Reply:
Makes sense. I can take a look at all the coffescript files this touches and push a fix.